### PR TITLE
Improve `Web/API/Document/images`

### PR DESCRIPTION
**Pull request by leon-win**
### Description

This PR updates `Web/API/Document/images`. List of changes:
- replace link to HTMLImageElement instead of constructor,
- improve example description,
- remove unnecesary line breaks.

### CODE REVIEW

File: files/en-us/web/api/document/images/index.md [hunk 1]
```diff
@@ -8,19 +8,15 @@ browser-compat: api.Document.images
...
-The **`images`** read-only property of
-the {{domxref("Document")}} interface returns a [collection](/en-US/docs/Web/API/HTMLCollection) of the [images](/en-US/docs/Web/API/HTMLImageElement/Image) in the current HTML document.
+The **`images`** read-only property of the {{domxref("Document")}} interface returns a [collection](/en-US/docs/Web/API/HTMLCollection) of the [images](/en-US/docs/Web/API/HTMLImageElement) in the current HTML document.
 
 ## Value
 
-An {{domxref("HTMLCollection")}} providing a live list of all of the images contained
-in the current document. Each entry in the collection is an
-{{domxref("HTMLImageElement")}} representing a single image element.
+An {{domxref("HTMLCollection")}} providing a live list of all of the images contained in the current document. Each entry in the collection is an {{domxref("HTMLImageElement")}} representing a single image element.
```
Comment (hamishwillee) [inline files/en-us/web/api/document/images/index.md:15]:
```suggestion
An {{domxref("HTMLCollection")}} providing a live list of all of the images contained in the current document.
Each entry in the collection is an {{domxref("HTMLImageElement")}} representing a single image element.
```

File: files/en-us/web/api/document/images/index.md [hunk 2]
```diff
@@ -8,19 +8,15 @@ browser-compat: api.Document.images
...
 
 ## Usage notes
 
-You can use either JavaScript array notation or the {{domxref("HTMLCollection.item", "item()")}} method on the returned collection to access the items in the collection.
-The following are equivalent:
+You can use either JavaScript array notation or the {{domxref("HTMLCollection.item", "item()")}} method on the returned collection to access the items in the collection. The following are equivalent:
```
Comment (hamishwillee) [inline files/en-us/web/api/document/images/index.md:19]:
```suggestion
You can use either JavaScript array notation or the {{domxref("HTMLCollection.item", "item()")}} method on the returned collection to access the items in the collection.
The following are equivalent:
```

**Review by hamishwillee — APPROVED** [review]
Thanks @leon-win 

I'll merge this one, but as a general rule MDN does not accept layout-only changes or "to my taste" changes on their own. The reason is that the changes do not affect the rendered content, but they do take time to review, and the time the reviewer team have is limited.

We do however accept technical fixes, and grammatical fixes etc that remove ambiguity, and so on. So you are more than welcome to add a technical fix, and as part of that also tidy layout.

**Comment by hamishwillee** [comment]
Ah, @leon-win Reading this more carefully, your change was technical - removing the constructor link. Apologies.

W.r.t. line breaks there are no firm rule on how these should be done in MDN. Most of the reviewer team strip out line breaks within a sentence. I tend to split on sentences.

**Comment by leon-win** [comment]
> Ah, @leon-win Reading this more carefully, your change was technical - removing the constructor link. Apologies.
> 
> W.r.t. line breaks there are no firm rule on how these should be done in MDN. Most of the reviewer team strip out line breaks within a sentence. I tend to split on sentences.

Hello, @hamishwillee! Thank you for review and comments!

Yes, I made two changes (about the constructor and about the example) and at the same time decided to remove line breaks))

I agree that this is a matter of taste.
In my opinion, using an autoformatter (if possible in this case) would eliminate ambiguity and reduce the burden on reviewers.

**Comment by hamishwillee** [comment]
> In my opinion, using an autoformatter (if possible in this case) would eliminate ambiguity and reduce the burden on reviewers.

@leon-win IMO the ambiguity only matters if it affects the rendered output, which it does not. IMO not worth the churn to do automatically. This is one of those things where people have strong opinions: I'd be pretty happy if we all agreed to break on sentences, but if the agreement ended up being "break on 80chars" I would cease working on MDN :-)

**Comment by leon-win** [comment]
> @leon-win IMO the ambiguity only matters if it affects the rendered output, which it does not. IMO not worth the churn to do automatically. This is one of those things where people have strong opinions: I'd be pretty happy if we all agreed to break on sentences, but if the agreement ended up being "break on 80chars" I would cease working on MDN :-)

The 80 character limit is more suitable for the Python language)) Therefore, stay in MDN :hugs: 

And about moving sentences to a new line. To be honest, I don’t really like this, precisely because these line breaks are not in the resulting display. And if we want to make a semantic emphasis, to separate parts of the text, then we need to make a break in the form of a separate paragraph, it seems to me so.

But now, seeing such line breaks, I will remember our conversation and try not to delete them =)

**Comment by hamishwillee** [comment]
Appreciate you not deleting them. I have several reasons for this approach.
1. Some years ago I was working with translators and they preferred sentence to paragraph breaks. That is probably tool-specific.
2. When reviewing in Github it is more likely to be able to have fixes constrained to a sentence than a paragraph - that just makes seeing the changes easier. Smaller lines obviously make this even easier, but I don't like those.
3. When I review, I read line by line, and it makes it easier for me to parse carefully. Again, this is a "me" thing.

I can live with paragraph breaks. There are many who are not convinced my approach is better.

**Pull request merged**